pkg/k2-asr/src/huggingface.py

```diff
@@ -41,17 +23,46 @@ def load_model(device="cpu", precision="fp32"):
     Returns:
       sherpa_onnx.OfflineRecognizer
     """
-    if precision not in HF_REPO_FILES:
+
+    if language == "ja":
+        hf_repo_id = "reazon-research/reazonspeech-k2-v2"
+        epochs = 99
+    elif language == "multi":
```

Comment:
I believe this is bilingual (English and Japanese) model, right?

Let's call it `en-ja`  or `ja-en` so that it won't get awkward when
we add more language support later on.